src/config/dynamic-theme-fixes.config

```diff
@@ -9477,6 +9477,23 @@ IGNORE INLINE STYLE
 
 ================================
 
+odysee.com
+
+INVERT
+.header__navigation-logo
+[data-vjs-player] .vjs-control-bar > [title="Autoplay Next Off"]
+[data-vjs-player] .vjs-control-bar > [title="Autoplay Next On"]::after
+
+CSS
+[dir] .content__viewer {
+    border: none;
+}
+[data-vjs-player] .vjs-control-bar .vjs-progress-control.vjs-control {
+    filter: invert(100%) hue-rotate(230deg) contrast(90%) !important;
```

Comment:
Hmm, why is their a custom filter? Is the `INVERT` not sufficient enough?

Reply:
No it `INVERT` does the job right,  but it turns this selector a little bit pale,  so i adjusted it a bit.

Reply:
i only changed the hue it's not that huge of a change

Reply:
>  that huge of a change

Correct, until we're going to add the factor of user settings which might have grayscale turned up all the way or sepia all over the place, in which this fix isn't going to match with the user, I'm trying to avoid such situations.

Reply:
Well, for now i don't mind removing this filter modification and leaving the default filter place, cause as i said it doesn't contribute much.